cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/trigger/ProgramStatusTriggerBuilder.java

```diff
@@ -57,11 +47,8 @@ public ProgramStatusTriggerBuilder(@Nullable String programNamespace, @Nullable
   @Override
   public ProgramStatusTrigger build(String namespace, String applicationName, String applicationVersion) {
```

Comment:
it can also be without namespace, i.e. different app in the same namespace as the deployed app